modules/gapi/src/backends/ie/giebackend.cpp

```diff
@@ -1057,10 +1087,9 @@ struct InferROI: public cv::detail::KernelTag {
 
                         IE::Blob::Ptr this_blob = extractBlob(*ctx, 1);
                         setBlob(req,
-                                ctx->uu.params.kind,
                                 *(ctx->uu.params.input_names.begin()),
-                                IE::make_shared_blob(this_blob,
-                                                     toIE(this_roi)));
+                                IE::make_shared_blob(this_blob, toIE(this_roi)),
+                                *ctx);
```

Comment:
Alignment.

Reply:
It's correct alignment now, `*ctx` not `make_shared_blob` argument